standard/docs/en/implementation/amendments.md

```diff
@@ -16,57 +16,169 @@ The nature of a change can be made explicit using:
+
+* **In the merged record**, the `releaseID` and `amendsReleaseID` fields from the `amendments` array can be used to lookup information in the `versionedRelease` section, to see where changes are explained by an amendment `rationale`. 
+
+### Example 2: Contract amendment
+
+This example shows an update to the value and scope of a contract.
+
+#### Contract
+
+A contract notice is published for the purchase of domestic appliances. The publisher builds a release and uses the 'contract' `tag`.
+
+See the JSON release below.
 
 ```eval_rst
 
-.. jsoninclude:: ../examples/merging/versioned.json
+.. jsoninclude:: ../_static/json/amendments-contract-example.json
+   :jsonpointer: /records/0/releases/0
+   :expand: tag, contracts, items
+   :title: Contract
+
+```
+
+#### Contract Amendment
+
+A few days after the contract release, its scope is increased to include the purchase of one additional appliance. A new 'contractAmendment' release is built, where a single item is added in the `contracts/items` block and the value of the contract is increased. A `amendments` block is included to explain the rationale of the changes. 
+
+See the example release below.
+
+```eval_rst
+
+.. jsoninclude:: ../_static/json/amendments-contract-example.json
+   :jsonpointer: /records/0/releases/1
+   :expand: tag, contracts, items, amendments
+   :title: ContractAmendment
+
+```
+
+Note that amendments can cover more than values or duration. Also, note that the publisher chose to not repeat the contract items, but add a new one with a new ID value. 
+
+In certain scenarios there may not be a valid `amendsReleaseID` and so it can be omitted, e.g. when historical data is being published in a single release.
+
+#### Record
+
+An example record for the whole process is shown below. Consider downloading the <a href="../../_static/json/amendments-contract-example.json" target="_blank">record example</a> and use the [Data Review Tool](https://standard.open-contracting.org/review) to explore the changes in the contracting process.
+
+Note that the `compiledRelease` contains all the items, included the latest one added in the contract amendment.
+
+```eval_rst
+
+.. jsoninclude:: ../_static/json/amendments-contract-example.json
    :jsonpointer: 
-   :expand: records, compiledRelease, tender, tag, versionedRelease, tender, value, amount, releaseTag, amendments
-   :title: versionedRecord
+   :expand: records, releases
+   :title: Record
 
 ```
 
+### Example 3: Amendments in a Easy Releases scenario
 
-Note in this example that:
+The Easy releases guidance (<mark>add link</mark>) explains how to build releases when there is no access to the change history for contracting processes. In a scenario when little or no change history is present, it may or may not be possible to identify and build amendments inside releases.
```

Comment:
```suggestion
The Easy releases guidance (<mark>add link</mark>) explains how to publish releases without storing or publishing a full change history. Depending on the source system, it may still be possible to publish a history of amendments when using this model.
```